modules/bmo-performing-a-live-update-to-the-hostfirmwaresettings-resource.adoc

```diff
@@ -34,37 +35,38 @@ $ oc patch hostfirmwaresettings <hostname> --type merge -p \ <1>
 +
 [NOTE]
 ====
-Get the `FirmwareSchema` resource to determine which settings the hardware supports. You can also use `oc edit <hostname> hostfirmwaresettings -n openshift-machine-api` to update the resource.
+Get the `FirmwareSchema` resource to determine which settings the hardware supports and what settings and values you can update. You cannot update read-only values and you cannot update the `FirmwareSchema` resource. You can also use `oc edit <hostname> hostfirmwaresettings -n openshift-machine-api` to update the resource.
 ====
 
-. Cordon the host by running the following command:
+. Cordon the node by running the following command:
 +
 [source,terminal]
 ----
-$ oc cordon <hostname>
+$ oc cordon <node_name> <1>
 ----
-+
-The Bare Metal Operator (BMO) updates the `operationalStatus` parameter to `servicing`.
+<1> Replace `<node_name>` with the name of the node. The Bare Metal Operator (BMO) updates the `operationalStatus` parameter to `servicing`.
 
-. Annotate the host to reboot by running the following command:
+. Power off the host for a period of 5 minutes by running the following command:
 +
 [source,terminal]
 ----
-$ oc annotate bmh <hostname> reboot.metal3.io=""
+$ oc patch bmh <hostname> --type merge -p '{"spec": {"online": false}}'
 ----
 +
-Once Ironic updates the resource, the BMO updates the `operationalStatus` parameter to `OK`.
+This step ensures that daemonsets or controllers can mark any infrastructure pods that might be running on the host as offline. Other hosts will handle incoming requests.
+
+. After 5 minutes, reboot the host by running the following command:
 +
-[NOTE]
-====
-Depending on the host hardware, the change might require more than one reboot. This might take 5 minutes or more.
-====
+[source,terminal]
+----
+$ oc patch bmh <hostname> --type merge -p '{"spec": {"online": true}}'
+----
 +
-If an error occurs, the BMO updates the `operationalStatus` parameter to `error` and retries the operation.
+During reboot, the servicing operation commences and the BMO sets the `operationalStatus` parameter to `servicing`. The BMO updates the `operationalStatus` parameter to `OK` after updating the resource. If an error occurs, the BMO updates the `operationalStatus` parameter to `error` and retries the operation.
```

Comment:
During boot 
(similar to my earlier comment - technically we're not rebooting since the node is powered off)